lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
```

Comment:
Yeah, that's true. The alternative is to make it a `--info` switch or include it in the `--version` output.